config.default

```diff
@@ -152,6 +152,11 @@ overwriteMpasClimatology = False
 #   'bilinear', 'neareststod' (nearest neighbor) or 'conserve'
 mpasInterpolationMethod = bilinear
 
+# the number of years per cached climatology file.  These cached files are
+# aggregated together to create annual climatologies, for example, when
+# computing the MOC.
```

Comment:
Maybe later as a separate PR.  But I also would be concerned about cluttering up `config.default` with stuff we won't ever change.  What is the use case you have in mind?